package/WindowsManaged/Program.cs

```diff
@@ -100,6 +100,30 @@ private static string DevolutionsWebClientPath
         }
     }
 
+    private static string LibXmfPath
+    {
+        get
+        {
+            string path = Environment.GetEnvironmentVariable("DGATEWAY_LIB_XMF_PATH");
+
+            if (string.IsNullOrEmpty(path) || !Directory.Exists(path))
```

Comment:
This should be `File.Exists`, not `Directory.Exists`. Probably the reason for your CI error.

Reply:
Nice catch! Copy paste gone wrong! 😂